src/prx/rinex_nav/test/benchmark.py

```diff
@@ -37,7 +35,7 @@ def generate_query(n_epochs=1):
 
 
 def benchmark(query, rinex_nav_file):
-    return rinex_nav_evaluate.compute_parallel(rinex_nav_file, query)
+    return rinex_nav_evaluate.compute(rinex_nav_file, query)
```

Comment:
What is the reasoning behind this change?

Reply:
Actually, there is a few cases where debugging is easier without parallelized code. And I also wanted to to check the gained time without parallelization. But I forgot to rollback the change. Thanks for noticing!